mods/d2k/maps/atreides-01a/atreides01a.lua

```diff
@@ -95,14 +96,15 @@ WorldLoaded = function()
 	SpiceToHarvest = ToHarvest[Difficulty]
 
 	InitObjectives(player)
-	KillAtreides = harkonnen.AddPrimaryObjective("Kill all Atreides units.")
-	GatherSpice = player.AddPrimaryObjective("Harvest " .. tostring(SpiceToHarvest) .. " Solaris worth of Spice.")
-	KillHarkonnen = player.AddSecondaryObjective("Eliminate all Harkonnen units and reinforcements\nin the area.")
+	KillAtreides = AddPrimaryObjective(harkonnen, "kill-atreides-units")
```

Comment:
```suggestion
	KillAtreides = AddPrimaryObjective(harkonnen, "")
```
and then remove the now unnecessary translations.